scripts/busybox.sh

```diff
@@ -47,6 +47,13 @@ while true; do
 done
 
 :	${usrdir:="${PWD}"/usr}
+
```

Comment:
oh, it work only  with makefile...it's just because i run it manually  without arg sry: )